### DISCUSSION TITLE
MonitorControl stopped working with Ventura 13.6.4 Beta

### DISCUSSION THREAD

**Discussion by mrgooge** [MonitorControl/MonitorControl#1483] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Nothing happens when standard brightness control buttons are pressed since installing Ventura Beta 13.6.4

### Steps to reproduce

press brightness key up or down

### Expected behavior

Worked fine adjusting brightness on Ventura 13.6.3

### Anything else?

I tried to uninstalled and reinstalled and also checked Security permissions.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.6.4 Ventura
- Mac model: iMac retina 5k
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): 
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Let's see if this issue is there by the RC as well.

**Comment by mrgooge** [comment]
Hey, sorry for the false report... I found the actual reason, I simultaneously installed MAC OS Ventura 13.6.4 and also Webroot SecureAnywhere software app.. I can resolve the issue by disabling the Webroot SecureAnywhere - Advanced setting - Realtime Shield - "Secure keyboard entry mode". That setting was disabling the keys from working in MonitorControl app. Thank you for your time!
